src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -56,36 +93,50 @@ public MessageObject getRestApiData(String restId) {
 	@Override
 	public MessageObject createRestApiData(RestAPI restApi) {
 		List<String> fileOfRest = new ArrayList<>();
-		
-		restApi.setRestId(_id+"");
-		_id = _id ++;
+		RestApi entity = new RestApi();
+
+		@SuppressWarnings("resource")
+		MongoClient mongoClient = new MongoClient("localhost", 27017);
```

Comment:
remove, contact me if you didn't understand what are you doing

Reply:
@SuppressWarnings("resource") will be removed